docs/pages/modules/config-plugin-and-native-module-tutorial.mdx

```diff
@@ -112,65 +146,70 @@ In another terminal window, compile and run the example app:
+> Although optional, use [`expo-module-scripts`](https://www.npmjs.com/package/expo-module-scripts) to simplify plugin development. It provides a recommended default configuration for TypeScript and Jest. For more information, see the [config plugins guide](https://github.com/expo/expo/tree/main/packages/expo-module-scripts#-config-plugin).
 
-Let's start by creating our plugin with this minimal boilerplate. This will create a **plugin** folder where we will write the plugin in TypeScript and add an **app.plugin.js** file in the project root, which will be the entry file for the plugin.
+Start creating your plugin with this minimal boilerplate. Create a **plugin** folder for writing the plugin in TypeScript and add an **app.plugin.js** file in the project root as the plugin's entry point.
 
-1. Create a **plugin/tsconfig.json** file:
+### Create a **plugin/tsconfig.json** file:
 
-   ```json plugin/tsconfig.json
-   {
-     "extends": "expo-module-scripts/tsconfig.plugin",
-     "compilerOptions": {
-       "outDir": "build",
-       "rootDir": "src"
-     },
-     "include": ["./src"],
-     "exclude": ["**/__mocks__/*", "**/__tests__/*"]
-   }
-   ```
+```json plugin/tsconfig.json
+{
+  "extends": "expo-module-scripts/tsconfig.plugin",
+  "compilerOptions": {
+    "outDir": "build",
+    "rootDir": "src"
+  },
+  "include": ["./src"],
+  "exclude": ["**/__mocks__/*", "**/__tests__/*"]
+}
+```
 
-2. Create a **plugin/src/index.ts** file for our plugin:
+### Create a **plugin/src/index.ts** file for your plugin:
 
-   ```ts plugin/src/index.ts
-   import { ConfigPlugin } from 'expo/config-plugins';
+```ts plugin/src/index.ts
+import { ConfigPlugin } from 'expo/config-plugins';
 
-   const withMyApiKey: ConfigPlugin = config => {
-     console.log('my custom plugin');
-     return config;
-   };
+const withMyApiKey: ConfigPlugin = config => {
+  console.log('my custom plugin');
+  return config;
+};
 
-   export default withMyApiKey;
-   ```
+export default withMyApiKey;
+```
 
-3. Finally, create an **app.plugin.js** file in the root directory. That will configure the entry file for our plugin:
+### Create an **app.plugin.js** file in the root directory:
 
-   ```js app.plugin.js
-   module.exports = require('./plugin/build');
-   ```
+```js app.plugin.js
+// This file configures the entry file for your plugin.
+module.exports = require('./plugin/build');
+```
 
-At the root of your project, run `npm run build plugin` to start the TypeScript compiler in watch mode. The only thing left to configure is our example project to use our plugin. We can achieve this by adding the following line to the **example/app.json** file.
+At the root of your project, run `npm run build plugin` to start the TypeScript compiler in watch mode. Next, configure your example project to use your plugin by adding the following line to the **example/app.json** file:
```

Comment:
Is the `plugin` part of the command? We never mention this before in this tutorial.

Reply:
Did you find out if this is a valid flag for this command? When I create an example project, I do not see the command being mentioned in the plugin's directory (maybe my understanding is wrong here):

![CleanShot 2024-11-26 at 17 32 42@2x](https://github.com/user-attachments/assets/015e0596-f95e-4231-908e-950949d80285)